libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -601,8 +644,10 @@ void CodeTransform::operator()(Block const& _block)
 	Scope* originalScope = m_scope;
 	m_scope = m_info.scopes.at(&_block).get();
 
-	int blockStartStackHeight = m_assembly.stackHeight();
+	int blockStartStackHeight = m_assembly.stackHeight() + static_cast<int>(m_delayedReturnVariables.size());
 	visitStatements(_block.statements);
+	if (!m_delayedReturnVariables.empty())
```

Comment:
Not needed.

Reply:
Maybe I should have squashed commits already...